src/config_test.go

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/newrelic/infra-integrations-sdk/data/metric"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestExcludeWildcard(t *testing.T) {
+	configYAML := `
+exclude:
+  metrics:
+    - "*"
+`
+	f, err := os.CreateTemp("", "cassandra_config.yml")
```

Comment:
Usually, we leverage https://pkg.go.dev/testing#T.TempDir since it is automatically removed. But you need to create a file in the directory, not sure what you prefer

Reply:
good point, I'll add it